subsys/bluetooth/shell/bt.c

```diff
@@ -1074,6 +1093,29 @@ static int cmd_scan_filter_set_addr(const struct shell *sh, size_t argc,
 	return 0;
 }
 
+static int cmd_scan_filter_set_rssi(const struct shell *sh, size_t argc, char *argv[])
+{
+	int err = 0;
+	long rssi;
+
+	rssi = shell_strtol(argv[1], 10, &err);
+
+	if (!err) {
+		if (IN_RANGE(rssi, INT8_MIN, INT8_MAX)) {
+			scan_filter.rssi = (int8_t)rssi;
+			scan_filter.rssi_set = true;
+			shell_print(sh, "RSSI cutoff set at %d dB", scan_filter.rssi);
+
+			return 0;
+		}
+	}
+
+	shell_print(sh, "value out of bounds (%d to %d)", INT8_MIN, INT8_MAX);
```

Comment:
We should perhaps also print the `err` here, as `-ERANGE` is just one of two possible return values.